tests/python/relay/test_op_level1.py

```diff
@@ -98,12 +96,12 @@ def test_unary_op(self, target, dev, relay_op, ref_func, dtype):
 
         if ref_func is not None:
             data = np.random.rand(*shape).astype(dtype)
-            ref_res = ref_func(data)
+            ref_res = ref_func(data).astype(dtype)
```

Comment:
Note that even the data is float16, some reference functions (e.g., `scipy.special.erf`) still return results in float32.